react/Button/index.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 import cx from 'classnames'
+import PropTypes from 'prop-types'
 import styles from './styles'
 
-export default (props) => {
-  const { theme = 'regular', busy, className, children, onClick } = props
+const Button = (props) => {
+  const { theme = 'regular', busy = false, className = '', children, onClick = () => { console.log('button was clicked') } } = props
```

Comment:
You forgot to remove the `console.log`. :)

Reply:
No default for onClick please :)

Reply:
No, I don't. That is a default value for the `onClick` function.

Reply:
Oh I get it, sorry. I let you two decide what's best then.